README.md

```diff
@@ -181,4 +181,6 @@ Feedstock Maintainers
 * [@bam241](https://github.com/bam241/)
 * [@gonuke](https://github.com/gonuke/)
 * [@scopatz](https://github.com/scopatz/)
+* [@katyhuff](https://github.com/katyhuff/)
+* [@FlanFlanagan](https://github.com/FlanFlanagan/)
```

Comment:
This is not correct.  They should be added to the `recipe/meta.yaml` file, and only there.